### PR TITLE
fix: fix scrollWidth for IE

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#3

### CODE REVIEW

File: src/Conveyer.ts [hunk 1]
```diff
@@ -297,8 +298,22 @@ class Conveyer extends Component<ConveyerEvents> {
...
+      const rect = scrollAreaElement.getBoundingClientRect();
+      const boundingSize = horizontal ? rect.width : rect.height;
+
+      // 0 < 0.5 && 0.5 < 1
+      if (offsetSize < boundingSize && boundingSize < offsetSize + 1) {
+        scrollSize = size;
+      }
+    }
+    this.size = size;
+    this.scrollSize = scrollSize;
```
Comment (WoodNeck) [inline src/Conveyer.ts:316]: I'm just being curious, why do you style protected values like public values?
I mean, those values are internal, not open to the users, so they should be treated like private values.
In short, how about adding underscore prefix(`_`) to those protected values/methods?

**Review by WoodNeck — APPROVED** [review]
LGTM

File: src/Conveyer.ts (hunk 1 above)
Comment (daybrush) [reply src/Conveyer.ts:316]: protected is not private even if used internally. protected can be converted to public. That's why I didn't use underscore. Discussion is needed.

Comment (WoodNeck) [reply src/Conveyer.ts:316]: Looks so.
I'm treating `protected` properties more like "private but can be inherited" (I think this is quite typical in OOP), so I don't get it about "protected can be converted to public".
Let's talk about it tomorrow :)

**Review by malangfox — APPROVED** [review]
LGTM

**Pull request merged**